### ISSUE TITLE
Unique image hash

### ISSUE THREAD

**Issue by drschwabe** [Lepovirta/kuvia#5]
Another feature request, each image could have it's own hash URL so you can link to them directly; ie- gallery.html#myimage.jpg

**Comment by jpallari** [comment]
Yup, that should be pretty easy to implement. I'll start working on it when I get the chance.

**Comment by jpallari** [comment]
BTW, would you prefer to have the path to the image as the hash or is the image index enough?

**Comment by drschwabe** [comment]
Start simple, image index would probably be just fine - key is just a unique URL that you can send people (ie- by email) that will take them to that specific image within the gallery.

**Comment by jpallari** [comment]
This will be fixed in PR #6. Can you pull the branch and try out the feature?

**Comment by drschwabe** [comment]
Pulled the branch, works great!  This is perfect.  
Also noticed the subtle style update - looks nice.

**Comment by jpallari** [comment]
Awesome. I'll make a release this week.

**Comment by drschwabe** [comment]
Just to nitpick, realizing this morning that image index won't necessarily suffice; at least not for my particular use case.   I have a gallery of images that I am continually adding to and as new images are uploaded (after running kuvia once to rebuild) I am sending an email to my client saying "hey this image is done, check it:  http://mysite.com/mygallery#3"  if the index is changing though as I add more the intended destination of those links will be incorrrect.   

So in a nutshell, filename based hash would be preferred!   :)

**Comment by jpallari** [comment]
Heh, the hash won't be very pretty thou. Image sources can contain full URLs which means that the full path to the image (whether its just a path or a URL) has to be in the hash. The implementation is pretty trivial thou. I'll check it out during the weekend.

**Comment by jpallari** [comment]
Hi, can you checkout the branch in pull request #7? Will that solve your use case?

**Comment by drschwabe** [comment]
Tested, works perfect !  Thanks.

**Comment by jpallari** [comment]
Cool. Merged #7.
